Multisig/UI/ClaimToken/TokenDistributionViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  TokenDistributionViewController.swift
+//  Multisig
+//
+//  Created by Mouaz on 9/5/22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class TokenDistributionViewController: UIViewController {
+
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var descriptionLabel: UILabel!
+    @IBOutlet private weak var nextButton: UIButton!
+    @IBOutlet weak var distributionView: BorderedCheveronButton!
+
+    private var onNext: (() -> ())?
+    private var stepNumber: Int = 1
+    private var maxSteps: Int = 3
```

Comment:
change it to 4

Reply:
updated, however it was being initialized with correct value in the init method